python/pyspark/sql/readwriter.py

```diff
@@ -742,10 +743,16 @@ def save(self, path=None, format=None, mode=None, partitionBy=None, **options):
     def insertInto(self, tableName, overwrite=None):
         """Inserts the content of the :class:`DataFrame` to the specified table.
 
+        Optionally overwriting any existing data.
+
         It requires that the schema of the :class:`DataFrame` is the same as the
         schema of the table.
 
-        Optionally overwriting any existing data.
+        Notes
+        -----
+        Unlike `saveAsTable`, `insertInto` ignores the column names and just uses
```

Comment:
Let's switch these to `` :meth:`DataFrameWriter.saveAsTable` ``, etc.

Reply:
Done